velox/exec/OutputBufferManager.h

```diff
@@ -36,7 +36,8 @@ class OutputBufferManager {
 
   /// When we understand the final number of split groups (for grouped
   /// execution only), we need to update the number of producing drivers here.
-  void updateNumDrivers(const std::string& taskId, uint32_t newNumDrivers);
+  /// Return true if the buffer exists for a given taskId, else returns false.
+  bool updateNumDrivers(const std::string& taskId, uint32_t newNumDrivers);
```

Comment:
Do we want to log an error or warning in its call sites? Thanks!

Reply:
It would be nice to check return value in call sites and ensure it is ok. For example, the Task may want to assert that it is already finished and not still running.